docs/root/configuration/statistics.rst

```diff
@@ -28,6 +28,7 @@ Server related statistics are rooted at *server.* with following statistics:
   parent_connections, Gauge, Total connections of the old Envoy process on hot restart
   total_connections, Gauge, Total connections of both new and old Envoy processes
   version, Gauge, Integer represented version number based on SCM revision
+  fips, Gauge, "1 if the server was built in a FIPS-compliant mode, 0 otherwise"
```

Comment:
I think I'd rather see this as something output in `envoy --version` or similar, instead of as a metric.

Reply:
LGTM. Definitely have it output as part of the admin endpoint also. I agree we want runtime access.

Reply:
Does this gauge still exist? Revert doc change?

Reply:
Yeah, it does.